CHANGE_LOG

```diff
@@ -213,7 +222,8 @@ CI/Infrastructure Updates:
 * PR #6252: Show channel URLs (Siu Kwan Lam)
 * PR #6338: Direct user questions to Discourse instead of the Google Group.
   (Stan Seibert)
-
+* PR #6474: Add skip on PPC64LE for tests causing SIGABRT in LLVM. (Stuart
+  Archibald)
```

Comment:
Think this perhaps ought to in ``Fixes``?

Reply:
Doesn't really fixes anything though. It's just a change is what is tested. That's why I think it is CI related

Reply:
I'm not sure not consistent with the categorisation of similar items, but probably not worth worrying about!